server/pulp/server/db/manage.py

```diff
@@ -191,6 +196,34 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        if status.get_workers():
+            last_worker_time = max([worker['last_heartbeat'] for worker
+                                    in status.get_workers()])
+            wait_time = timedelta(seconds=constants.MIGRATION_WAIT_TIME) -\
+                (UTCDateTimeField().to_python(datetime.now()) - last_worker_time)
```

Comment:
Can this expression be assigned to a variable to lower the complexity of this L203-L204. 1-liners like this are hard to understand and benefit from bring broken up. Eric Helms taught me that when statements are small, and variable names are really clear, readability goes way up and comments are almost never needed.